packages/core-flows/src/inventory/steps/deatach-inventory-items.ts

```diff
@@ -13,47 +13,21 @@ export const deatachInventoryItemStep = createStep(
   async (ids: string[], { container }) => {
     const remoteLink = container.resolve(ContainerRegistrationKeys.REMOTE_LINK)
 
-    const linkModule: ILinkModule = remoteLink.getLinkModule(
-      Modules.PRODUCT,
-      "variant_id",
-      Modules.INVENTORY,
-      "inventory_item_id"
-    )
+    await remoteLink.delete({
```

Comment:
oh cool, didn't know this does a soft delete. Would be good to have a test that checks this.

Reply:
suggestion: don't have to do this now, but I wonder if we can do a generic `deleteLinkStep()` that accepts an object. Something like this:

```
deleteLinkStep({
  [Modules.INVENTORY]: { inventory_item_id: data.ids },
})
```

Reply:
https://github.com/medusajs/medusa/blob/94208a979bb14a6bbbb4ea5ba8f712a7068917df/packages/core-flows/src/common/steps/remove-remote-links.ts

if needed you can bring this one to the PR

Reply:
nice!

Reply:
@carlos-r-l-rodrigues very nice!